runtime/common/marshaling/index.ts

```diff
@@ -113,12 +184,12 @@ export function wrapError(err: Error): Error {
   //     'to actually fetch from the remote API.';
   // }
 
-  return new Error(marshalValue(err));
+  return new Error(v8.serialize(marshalValue(err)).toString('base64'));
```

Comment:
in the worst case, just using JSON for this case. if an error object is 2MB, we are screwed. lol